src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java

```diff
@@ -147,16 +151,18 @@ public void testValidate() throws Exception {
 
         errorMessage = String.format(FieldValidator.PARTICIPANT_TYPE_TEAM_ERROR_MESSAGE,
                                      fq.recipientType.toDisplayRecipientName(),
-                                     fq.giverType.toDisplayGiverName()) + EOL
+                                     fq.giverType.toDisplayGiverName()) + System.lineSeparator()
                        + String.format(FieldValidator.PARTICIPANT_TYPE_ERROR_MESSAGE, fq.showGiverNameTo.get(0).toString(),
-                                       FieldValidator.VIEWER_TYPE_NAME) + EOL
-                       + "Trying to show giver name to STUDENTS without showing response first." + EOL
+                                       FieldValidator.VIEWER_TYPE_NAME) + System.lineSeparator()
+                       + "Trying to show giver name to STUDENTS without showing response first."
+                       + System.lineSeparator()
                        + String.format(FieldValidator.PARTICIPANT_TYPE_ERROR_MESSAGE,
                                        fq.showRecipientNameTo.get(0).toString(),
-                                       FieldValidator.VIEWER_TYPE_NAME) + EOL
-                       + "Trying to show recipient name to STUDENTS without showing response first." + EOL
+                                       FieldValidator.VIEWER_TYPE_NAME) + System.lineSeparator()
+                       + "Trying to show recipient name to STUDENTS without showing response first."
+                       + System.lineSeparator()
```

Comment:
No need to writing it on a separate line

Reply:
Sir, if I am writing it in the same line then it increases line limit of 125 columns